src/node.cc

```diff
@@ -3637,6 +3637,8 @@ static void ParseArgs(int* argc,
   const char** new_v8_argv = new const char*[nargs];
   const char** new_argv = new const char*[nargs];
   const char** local_preload_modules = new const char*[nargs];
+  bool use_bundled_ca = false;
+  bool use_openssl_ca = false;
```

Comment:
I generally find it more readable but that's just me ;-) .. I know there are broadly different opinions about the use of bitfields ;-)